db/db_basic_test.cc

```diff
@@ -2104,14 +2142,20 @@ class DBBasicTestWithTimestampWithParam
 TEST_P(DBBasicTestWithTimestampWithParam, PutAndGet) {
   const int kNumKeysPerFile = 8192;
   const size_t kNumTimestamps = 6;
-  bool memtable_only = GetParam();
+  const size_t kNumKeysPerTimestamp = (kNumKeysPerFile - 1) / kNumTimestamps;
+  const size_t kSplitPosBase = kNumKeysPerTimestamp / 2;
+  int flush_compaction = GetParam();
```

Comment:
The meaning of `flush_compaction` can be clearer?

Reply:
Will refactor.